ontopbm-aquatic/.htaccess

```diff
@@ -1,5 +1,5 @@
 Header set Access-Control-Allow-Origin *
 Header set Access-Control-Allow-Headers DNT,X-Mx-ReqToken,Keep-Alive,User-Agent,X-Requested-With,If-Modified$
 Options +FollowSymLinks
-RewriteEngine on
+RewriteEngine On
```

Comment:
I'm guessing you tested this?  Docs just use lowercase `on`:
https://httpd.apache.org/docs/2.4/mod/mod_rewrite.html#rewriteengine

Reply:
It was an oversight, should be lowe case